locales/fr-FR.json

```diff
@@ -161,7 +162,7 @@
     "blocked_domains": "Domaines bloqués",
     "blocked_users": "Utilisateur·ice·s bloqué·e·s",
     "bookmarks": "Marque-pages",
-    "built_at": "Dernier build {0}",
+    "built_at": "Dernière build {0}",
```

Comment:
For me build is masculine.
Le build and not la build.

Reply:
Although I can't find an explicit definition anywhere, there are more occurences of "la build" than "le build" on the web in the context of computing.

I'd argue that since "build" can be translated with "compilation" or "construction" which are both feminine words then it sounds more natural for "build" to be feminine as well.

Thinking about it, I instead replaced "build" with "compilation" in order to simplify that decision.